frontend/src/tests/components/Commons/ManageCowsModal.test.js

```diff
@@ -32,6 +32,22 @@ describe('ManageCowsModal', () => {
     expect(screen.getByTestId('buy-sell-cow-modal')).toBeInTheDocument();
   });
 
+  test('modal has correct buy capitalization', () => {
+    render(
+        <ManageCowsModal isOpen={true} onClose={mockOnClose} message="buy" setNumber={mockSetNumber} />
+    );
+
+    expect(screen.getByText('Buy')).toBeInTheDocument();
+  });
+
+  test('modal has correct sell capitalization', () => {
+    render(
+        <ManageCowsModal isOpen={true} onClose={mockOnClose} message="sell" setNumber={mockSetNumber} />
+    );
+
+    expect(screen.getByText('Sell')).toBeInTheDocument();
+  });
+
   test('does not render the modal when isOpen is false', () => {
     render(
         <ManageCowsModal isOpen={false} onClose={mockOnClose} message="buy" setNumber={mockSetNumber} />
```

Comment:
And similarly, here, though its sort of a moot point.

```suggestion
        <ManageCowsModal isOpen={false} onClose={mockOnClose} message="Buy" setNumber={mockSetNumber} />
```

Reply:
resolving because of this:
https://github.com/ucsb-cs156-f24/proj-happycows-f24-10/pull/14#issuecomment-2480151773